portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import java.util.Arrays;
+
+public class Comment{
+
+    private String name;
+    private String content;
+    
+    public Comment(String name, String content){
+        this.name = name;
+        this.content = content;
+    }
+
+    public String getName(){
+        return this.name;
+    }
+
+    public void setName(String newName){
+        name = newName;
```

Comment:
Nit (btw when comments start with that it usually means that this is a kind of nitpicky/meh comment): Its ever so slightly more clear what is happening when you use `this.name = ... `

Reply:
Completed!